### PR TITLE
[WIP] PurpleAir Layer 

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
Solves #14  💜 !
🔜

### CODE REVIEW

**Comment by sagarpreet-chadha** [comment]
@jywarren 
used this for heat maps : https://www.patrick-wied.at/static/heatmapjs/

screen shot 2018-06-22 at 5 40 41 pm

**Comment by jywarren** [comment]
Ooooooooo! Very nice!!!!

**Comment by sagarpreet-chadha** [comment]
Most probably we have to convert PM2.5 Value to US-based AQI Value , using this : 

Asking on _publiclab.org_ 🎈  .

image

File: src/purpleLayer.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              var lat = data.Lat ;
+              var lng = data.Lon;
+              var value = parseFloat(data.PM2_5Value) ;  //PM2.5 VALUE in microgram per metre cube
+
+              var purpleLayer_object = new Object() ; 
+              purpleLayer_object.lat = lat ;
+              purpleLayer_object.lng = lng ;
+
+              var aqi ;
+              if(value>=0 && value<=12.0)
```
Comment (sagarpreet-chadha) [inline src/purpleLayer.js]: @jywarren ...used the formula given on Wikipedia page !
The AQI value calculated matches with the one calculated here : https://www.purpleair.com/map?&zoom=4&lat=39.96179960675426&lng=-74.31505939400137&clustersize=45&orderby=L&latr=46.230881869445966&lngr=126.56250000000006   🎉 😄  .

**Comment by sagarpreet-chadha** [comment]
Looks like this now : 

screen shot 2018-06-23 at 12 29 17 pm

**Comment by sagarpreet-chadha** [comment]
Hi @jywarren 

How does this look ?

**Comment by jywarren** [comment]
It looks great. Are there also individual readings available at specific stations? (maybe a separate issue)

Also, is there labeling or links we could provide to explain where the data comes from, what time period it corresponds to, things like that?

Awesome work! 👍 👍 👍

**Comment by sagarpreet-chadha** [comment]
Thanks!

Yes , for each specific station we can show individual reading and timestamp and many other things , but there seems to be no way we can show them on Heat Map . 

We can make another layer with markers ❓ 

And should i merge this then ❓

**Comment by jywarren** [comment]
Yes, perhaps we should have two layers then - the individual markers, and
the heat map. Yes, go ahead! :-)

**Comment by sagarpreet-chadha** [comment]
@jywarren ...merging  👍 👍 👍

**Pull request merged**